LOSS/guides/Demonology/Spec Mechanics.html

```diff
@@ -1,29 +1,94 @@
 <center>
     <img class="heading" src="http://i.imgur.com/zGeOXY9.png">
-</center><span style="font-size:20px"><strong><u>Demonwrath</u></strong></span> There is a lot of confusion out there about how Demonwrath actually works but in reality it is very simple;
+</center>
+<span style="font-size:20px"><strong><u>Demonwrath</u></strong></span>
```

Comment:
Moving forward we should get rid of these "header" spans and Instead use h2-h6.
h2 - For Section Headings
h3 - Sub Section Headings